docs/install.md

```diff
@@ -159,10 +171,10 @@ pip install -v -e .
 pip install "mmcls<0.23.1 " "mmtrack<0.13.0"
```

Comment:
mmtrack 0.12.0 supports mmcv 1.5.0 at most. If 0.13.0 works, we'd better install a newer one.
See details at:
https://github.com/open-mmlab/mmtracking/blob/v0.12.0/mmtrack/__init__.py

Reply:
@ttxskk I believe the current setting is modified by #228. It seems that mmtracking v0.12.0 can run on mmcv 1.5.3?

Reply:
We should use mmtracking v0.13.0.

Reply:
Then it's `pip install git+https://github.com/open-mmlab/mmtracking.git@v0.13.0`.